src/python/bot/untrusted_runner/untrusted.py

```diff
@@ -215,9 +215,7 @@ def _get_tls_cert_and_key():
   # TODO(mbarbella): Remove this after migrating to Python 3. The grpc library
   # has explicit type checks against str.
   cert_contents = str(compute_metadata.get('instance/attributes/tls-cert'))
-  cert_contents = utils.newstr_to_native_str(cert_contents)
```

Comment:
This changes the type from "bytes" to "str. Can you fix this such that it looks like:

```python
cert_contents = compute_metadata.get('instance/attributes/tls-cert').encode()
...
```

I believe the `str` is no longer necessary either.

Reply:
Thanks, Updated in the PR. Please check!